murmurhash3.js

```diff
@@ -0,0 +1,65 @@
+/* eslint no-fallthrough: 0, default-case: 0 */
```

Comment:
Oh and I just this library because it has less change of collisions then the hashCode function I showed before: http://stackoverflow.com/a/22429679/1660118